x/msg_authorization/types/codec.go

```diff
@@ -15,10 +18,34 @@ func RegisterCodec(cdc *codec.Codec) {
 	cdc.RegisterInterface((*AuthorizationI)(nil), nil)
 }
 
+// RegisterInterfaces will registers the msgs and interfaces of the msg_auth module
+func RegisterInterfaces(registry types.InterfaceRegistry) {
+	registry.RegisterImplementations((*sdk.Msg)(nil),
+		&MsgGrantAuthorization{},
+		&MsgRevokeAuthorization{},
+		&MsgExecAuthorized{},
+	)
+
+	// registry.RegisterImplementations(
+	// 	(*AuthorizationI)(nil),
+	// 	&SendAuthorization{},
+	// 	&AuthorizationGrant{},
+	// 	&GenericAuthorization{},
+	// )
+
+	registry.RegisterInterface(
+		"cosmos_sdk.msgauth.v1.msgauth",
```

Comment:
This should be `AuthorizationI` right?

```suggestion
		"cosmos_sdk.msgauth.v1.AuthorizationI",
```